be/src/vec/exec/format/parquet/schema_desc.cpp

```diff
@@ -291,6 +299,11 @@ std::pair<DataTypePtr, bool> FieldDescriptor::convert_to_doris_type(
     } else if (logicalType.__isset.TIMESTAMP) {
         ans.first = DataTypeFactory::instance().create_data_type(
                 TYPE_DATETIMEV2, nullable, 0, logicalType.TIMESTAMP.unit.__isset.MILLIS ? 3 : 6);
+    } else if (logicalType.__isset.JSON) {
```

Comment:
is JSON a new type that we do not support before?

Reply:
before some type not in switch case will throw exception and ignored.
so will all convert into TYPE_STRING